omod/src/main/webapp/resources/htmlforms/096a-HealthUnitTBRegister-SummaryPage.xml

```diff
@@ -899,7 +899,7 @@
       </table>
     </fieldset>
 
-    <enrollInProgram programId="9dc21a72-0971-11e7-8037-507b9dc4c741" />
+	  <postSubmissionAction class="org.openmrs.module.aijar.htmlformentry.TBProgramEnrollmentPostSubmissionAction" />

```

Comment:
I suggest we name this action to TBProgramPostSubmissionAction since it deals with both enrollment and exit